src/EventListener/QueryCreatedEventInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/*
+
+This file was created by developers working at BitBag
+
+Do you need more information about us and what we do? Visit our   website!
+
+We are hiring developers from all over the world. Join us and start your new, exciting adventure and become part of us: https://bitbag.io/career
+  */
+
+namespace BitBag\SyliusElasticsearchPlugin\EventListener;
+
+use Elastica\Query\AbstractQuery;
+use ECSPrefix20211002\Psr\EventDispatcher\StoppableEventInterface;
```

Comment:
Are you sure that this is a correct import? :)

Reply:
Used this one coz in case `Psr\EventDispatcher\StoppableEventInterface;` symfony 4.4 throw error about wrong namespace and cant find interface.